main.go

```diff
@@ -108,19 +126,21 @@ func main() {
 	}
 	kingpin.Parse()
 
-	showGreeting()
-
+	// Assign every global variable first
 	_isDryRun = *args.dryRun
 	_verbose = *args.verbose
 
+	// Initiate logging
+	initLogging()
```

Comment:
I need to move logging below parsing the command argument, because I need the `_verbose` variable to be set first. In order to know what kind of logging level should the program use.